### PR TITLE
[OYPD-410-354] Add some basic styles for the 404 and 403 pages.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Issue #354 
Drupal.org issue: https://www.drupal.org/node/2878950

Steps
- [x] go to [sitename]/about1111
- [x] vefify 404 page have style as here https://cloud.githubusercontent.com/assets/1316234/26104495/c5e865aa-3a46-11e7-9d31-b65184def5a0.png
- [x] go to page [sitename]/user/logout
- [x] vefify 403 page have style as here 
https://cloud.githubusercontent.com/assets/1316234/26104519/e10469ec-3a46-11e7-87d4-623c9b9788ab.png

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1196 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1196 |
| Installation process | http://install.openy-dev.ffwua.com/build1196/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1196/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1196/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

File: themes/openy_themes/openy_rose/templates/page/page--403.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "page.html.twig" %}
+
+{% block pagecontent %}
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h1>403 Access Denied</h1>
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/page/page--403.html.twig]: Should be translatable.

File: themes/openy_themes/openy_rose/templates/page/page--403.html.twig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <p>You are not authorized to access this page.</p>
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/page/page--403.html.twig]: Should be translatable.

File: themes/openy_themes/openy_rose/templates/page/page--404.html.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% extends "page.html.twig" %}
+
+{% block pagecontent %}
+  <div class="container">
+    <div class="row error-page">
+      <div class="col-md-12">
+        <h1>404 Page not found</h1>
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/page/page--404.html.twig]: Should be translatable.

File: themes/openy_themes/openy_rose/templates/page/page--404.html.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        <p>Oops! The page that you're looking for was not found.</p>
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/page/page--404.html.twig]: Should be translatable.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1199 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1199 |
| Installation process | http://install.openy-dev.ffwua.com/build1199/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1199/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1199/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1200 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1200 |
| Installation process | http://install.openy-dev.ffwua.com/build1200/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1200/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1200/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**